_projects/molssi.md

```diff
@@ -4,9 +4,25 @@ collection: projects
 permalink: /projects/molssi
 abstract: "Enabling science, education, and cooperation to serve the worldwide community of computational molecular scientists."
 status: active
+logo: molssi_logo.png
 ---
 
-The Molecular Sciences Software Institute serves as a nexus for science, education, and cooperation serving the worldwide community of computational molecular scientists – a broad field including of biomolecular simulation, quantum chemistry, and materials science. The Institute is responsible for significant advances in software infrastructure, education, standards, and best-practices that are needed to enable the molecular science community to open new windows on the next generation of scientific Grand Challenges, ranging from the simulation of intrinsically disordered proteins associated with a range of diseases to the design of new catalysts vital to the global chemical industry and climate change. The RADICAL Lab is playing a leading role in the software engineering activities and middleware design and development upon which MolSSI is developing its capabilities.
-
-<a href="https://molssi.org"><i></i>Website</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1547580"><i class="fa fa-nsf"></i>NSF Award</a>
\ No newline at end of file
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/molssi_main_logo.png" width="150">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```